TypeCobol/Compiler/CupParser/NodeBuilder/ProgramClassBuilder.cs

```diff
@@ -210,6 +212,33 @@ public virtual void StartCobolCompilationUnit()
 
         public virtual void StartCobolProgram(ProgramIdentification programIdentification, LibraryCopyCodeElement libraryCopy)
         {
+            // check if a dot follows PROGRAM-ID
```

Comment:
This control only rely on Tokens of the CodeElement.
It must be moved to a CodeElement checker.
Otherwise in incremental mode, it'll be done even if the line which contains PROGRAM-ID doesn't change.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1651/commits/5e258c0cb8c42152413b25cb6ba82ae7bea386d0

Reply:
Fix compute of the program names in Helpers.LoadDependencies: see https://github.com/TypeCobolTeam/TypeCobol/pull/1651/commits/fe8c715e55817504ee21d2cb9b8c7ebbbe6c1678